src/scripts/install.sh

```diff
@@ -1,4 +1,4 @@
-if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
+if [ "$ID" = 0 ]; then export SUDO=""; else export SUDO="sudo"; fi
```

Comment:
Rather than changing this, maybe we should ensure this check is just not ran on Alpine, where I believe the issue came from. We could use ID only in the case of Alpine.

Reply:
Added validation: 

```
if cat /etc/issue | grep "Alpine" >/dev/null 2>&1; then
    if [ "$ID" = 0 ]; then 
        export SUDO=""; 
    else 
        export SUDO="sudo"; 
    fi
else 
    if [[ $EUID == 0 ]]; then 
        export SUDO=""; 
    else 
        export SUDO="sudo"; 
    fi
fi
```